### ISSUE TITLE
Augustus and using RNAseq reads exclusively

### ISSUE THREAD

**Issue by gchchung** [Gaius-Augustus/BRAKER#667]
Question related to issue [383](https://github.com/Gaius-Augustus/BRAKER/issues/383). 

How does one interpret this description on how to make the malus in `rnaseq.cfg` so that Augustus does not predict _any_  introns not supported by RNAseq?

Per the original `extrinsic.cfg`:

> Increase the bonus to make AUGUSTUS obey more hints, decrease the malus to make AUGUSTUS predict few features that are not supported by hints.

Does this statement mean "**_Decrease_** the malus to make AUGUSTUS predict **_fewer_** features that are not supported by hints"? Or does this statement mean "**_Decrease_** the malus to make AUGUSTUS predict **_a few_** features that are not supported by hints"? The first interpretation means the smaller the malus, the fewer unsupported features. The second interpretation means the smaller the malus, you _**allow**_ a few features that are not supported by hints.

Asking as I've tried using the BRAKER2 pipeline to do gene prediction using Illumina read alignments. While most of the time, Augustus follows the read alignments very closely, sometimes it can invent a wholly new exon/intron out of the blue. Shown below in the red box is a clearly unsupported intron by Augustus (the 'g19940', bottommost track), while StringTie (the 'STRGs') follows the read data, but incorrectly joins the a gene next door.
![image](https://github.com/Gaius-Augustus/BRAKER/assets/69369525/8fec4606-0027-4615-ac1f-746247999188)

I understand BRAKER2 and its components are really meant for protein coding gene prediction, so while I'm annotating this new nematode genome, I'm hoping to look for a very specific non-coding transcript I expect to be very rare. Augustus seems to consistently outperform StringTie in calling the correct gene boundaries, but it calls introns sometimes where no reads support them. I do not want any read-supported transcript sequences to be cut out by Augustus at all. I suspect all I will need to change is this line in the `rnaseq.cfg`, esp the 0.34 or 1 after 'intron'.

intron      1        0.34       M       1  1e+100      RM       1       1       E       1     1e6       W       1       1       P       1       1

How do I change this line so that Augustus does not predict any introns not supported by reads?

**Comment by gchchung** [comment]
@KatharinaHoff Thanks for your response. In this specific instance, the Augustus splice isn't supported by spliced reads. I think the pileup of reads in the red box are either from the 3' UTR of the gene immediately to the left, or a highly transcribed ncRNA that overlaps somewhat with the 3' UTR.

In one of several test runs, I set the malus to 0 such that the line from my original post (from `rnaseq.cfg`) was changed to

intron      1           **0**       M       1  1e+100      RM       1       1       E       1     1e6       W       1       1       P       1       1

I can happily report that this stopped Augustus from predicting most introns that are unsupported by read hints. As shown below, the gene models from the same region now appear to be more consistent with the RNAseq data.

![image](https://github.com/Gaius-Augustus/BRAKER/assets/69369525/f9f36198-5b4f-463c-b90f-86b25a6035c1)

However, it did not completely prohibit prediction of all features without evidence, but I think most of these features can be checked against the FPKM values with a short custom script.

As for the rare ncRNA, I stuck with the StringTie transcriptome. Cross referencing with the Augustus predictions above, I was able to easily eliminate StringTie transcripts that are coding, and I found some cool ncRNA candidates to investigate.

TL;DR set the malus to 0 and most introns unsupported by reads seem to disappear.

Thanks!
